index.d.ts

```diff
@@ -0,0 +1,20 @@
+export interface UTXO {
+    txid: string | Buffer,
+    vout: number,
+    value: number,
+    nonWitnessUtxo? : Buffer,
+    witnessUtxo? : {
+        script: Buffer,
+        value: number
+    }
+}
+export interface Target {
+    address: string,
+    value: number
```

Comment:
`value`  should be optional (in this case, library assings all remaining funds to this target)